tools/README.md

```diff
@@ -1,64 +1,65 @@
-# Tools for arti-cache
+# Tools for generating custom directory information
 
-## Overview
+To reduce bandwidth use of lightarti-rest, the library relies on custom Tor directory information. The scripts in the `tools/` directory generate the required files, that can then be downloaded by apps for use with lightarti-rest.
 
-To send some data anonymously over the network, Lightarti-rest relies on a
-subset of relays of the Tor network.
+The scripts in this directory aim to pick reliable nodes so that ideally the consensus can be used for up to a week. We refer to the [accompanying Jupyter notebook](churn_analysis.ipynb) for a historical analysis of churn on the Tor network that shows shows that this approach is viable.
```

Comment:
```suggestion
The scripts in this directory pick reliable nodes so that ideally the consensus can be used for up to a week. Reliability is measured as INSERT MEASURES HERE. We refer to the [accompanying Jupyter notebook](churn_analysis.ipynb) for a historical analysis of churn on the Tor network that shows shows that this approach is viable.
```